src/inscriptions/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { RequestOptions, RequestPayload } from '../types';
+
+export interface CreateInscriptionPayload extends RequestPayload {
+  contentType: string;
+  content: string;
+  payloadType: 'PLAIN_TEXT' | 'BASE_64';
+  inscriptionFee?: number;
```

Comment:
Might be more appropriate to name this `appFee` so that it's not confused with the inscription fee set by the inscription service.